umm-lib/resources/schema/iso_mends/resources/Codelist/eosCodelists.xml

```diff
@@ -187,4 +187,53 @@
          </gmx:codeEntry>
       </gmx:CodeListDictionary>
    </gmx:codelistItem>
+   <!--=== DS_AssociationTypeCode ===-->
```

Comment:
For clarification, what do the XML changes have to do with linter issues?

Reply:
Nothing.  It looks like the code from my previous issue that I was working got into this PR.  I will need to redo my changes and issue a new PR.